### PR TITLE
Add a remove frontend command

### PR DESCRIPTION
**Pull request by loic425**

### CODE REVIEW

File: bin/remove-frontend [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+$data = <<<EOM
+app_frontend_homepage:
+    path: /
+    defaults:
+        _controller: FrameworkBundle:Redirect:redirect
+        route: app_backend_dashboard
+
+EOM;
+
+file_put_contents($projectDir.'/config/routes/frontend.yaml', $data);
```
Comment (gabiudrescu) [inline bin/remove-frontend]: why don't you use some functions from Symfony FileSystem, if you have it already instantiated. eg: dump() or appendToFile() ?

Comment (gabiudrescu) [reply bin/remove-frontend]: https://symfony.com/doc/current/components/filesystem.html#dumpfile

File: README.md [hunk 2]
```diff
@@ -19,13 +19,21 @@ Install php dependencies using composer
...
+Install project
 ```bash
 $ bin/console app:install
 $ yarn install && yarn build (or "yarn dev" for development usage)
 $ symfony server:start --no-tls
 ```
 
+Remove frontend (optional)
+
+When you don't need any frontend on your app with Symfony, this commands remove frontend part of Monofony.
```
Comment (gabiudrescu) [inline README.md]: I would rephrase this to avoid any confusion with frontend = node part.

my take would be:

> Sometimes, your application does not need a public facing section. You just need the admin and everybody should be logged in before accessing that. This is what we call in Monofony „frontend”. 
> 
> When you don't need any frontend on your app with Symfony, this commands remove frontend part of Monofony.

Or rephrase entirely, without using frontend anymore. wdyt?

File: bin/remove-frontend (hunk 1 above)
Comment (loic425) [reply bin/remove-frontend]: Just don't know this part. But I replace totally the content (no append option).
I can try dumpfile.

File: README.md (hunk 2 above)
Comment (loic425) [reply README.md]: > When building a SPA with Angular/React/VueJs or a mobile app, you may just need the administration panel and/or the API.
> This command basically leaves you only with the admin, protected routes and the API.
> The public part https://monofony.mobizel.com/ here is removed an redirected to the administration panel.

Comment (gabiudrescu) [reply README.md]: this new version sounds much better.
